docs/ROADMAP.md

```diff
@@ -0,0 +1,15 @@
+# Amethyst Roadmap 2019
+
+## Current Features:
+
+- 2D/3D Rendering
```

Comment:
Can we have each of those be categories? Like

```markdown
# Amethyst Roadmap

## Rendering

### Current features

* implemented technique 1
* implemented technique 2
* point lights, directional lights, etc

### Planned features

* Shadows
* 3D instancing
```

Reply:
I purposely kept it without categories because I want this to be really simple and easy to read. We have a more complex roadmap that will be avail but this is for people that want to get a very high-level view.